core/modules/system/js/status.js

```diff
@@ -1,13 +1,11 @@
 (function ($) {
 
 /**
- * Filters the module list table by a text input search string.
- *
- * Additionally accounts for multiple tables being wrapped in "package" fieldset
- * elements.
+ * Toggles the more/less links that show/hide details in the "Status report"
+ * page.
  */
 Backdrop.behaviors.statusReport = {
-  attach: function(context, settings) {
+  attach: function() {
```

Comment:
This changes the JS on the status report page -- is that intentional? The rest of this PR is for the system updates form, so I suspect these changes might be from another issue?

Reply:
Yes, it is intentional. `context` and `settings` are never used anywhere in that function in status.js - they must have been introduced as a copy-paste from other code.

Reply:
PS: you will notice that the docblock in the `status.js` file mentions the modules list table (which I also fix in this PR), so there was definitely some copy-paste involved when this was introduced.

Reply:
PPS: once this PR gets merged, we'll have a total of 3 places where we are using custom JS to provide this more/less functionality, and all that JS is ~90% similar. I've filed https://github.com/backdrop/backdrop-issues/issues/5090 to consider standardize that UI pattern, and use `<details>` instead.